web-common/src/features/dashboards/pivot/pivot-infinite-scroll.ts

```diff
@@ -115,14 +116,7 @@ export function getColumnFiltersForPage(
     effectiveColumnsPerPage,
   );
 
-  const colFilters: MetricsViewFilterCond[] = [];
-
-  Object.entries(pageGroups).forEach(([colDimensionName, values]) => {
-    colFilters.push({
-      name: colDimensionNames[parseInt(colDimensionName)],
-      in: Array.from(values),
-    });
-  });
-
-  return colFilters;
+  return Object.entries(pageGroups).map(([colDimensionName, values]) =>
+    createInExpression(colDimensionName, Array.from(values)),
```

Comment:
The variable name is mistyped leading to an error while adding row dimensions. Can be changed to - 

 ```
  return Object.entries(pageGroups).map(([colDimensionId, values]) =>
    createInExpression(
      colDimensionNames[parseInt(colDimensionId)],
      Array.from(values),
    ),
  );
```